pkg/base1/cockpit.js

```diff
@@ -1068,6 +1071,31 @@ function basic_scope(cockpit, jquery) {
         return format_units(number, byte_suffixes, factor, separate);
     };
 
+    cockpit.get_byte_units = function get_byte_units(guide_value, factor) {
```

Comment:
This needs documentation. How about calling it 'parse_byte_units' to match with the 'format_bytes' naming?